src/matlab2tikz.m

```diff
@@ -204,6 +204,10 @@ function matlab2tikz(varargin)
     ipp = ipp.addParamValue(ipp, 'showWarnings', true, @islogical);
     ipp = ipp.addParamValue(ipp, 'checkForUpdates', isempty(VCID), @islogical);
 
+    ipp = ipp.addParamValue(ipp, 'semanticalLineWidth', false, @islogical);
+    ipp = ipp.addParamValue(ipp, 'semanticalValueList', [.1,.2,.4,.6,.8,1.2,1.6], @isnumeric);
```

Comment:
@egeerardyn I would generalize that checking function to apply for any [Name, Value] list. Basically this would just be a rename to isValidOptionList or whatever and update the comment of the second line